thoth/package_extract/image.py

```diff
@@ -546,6 +546,50 @@ def get_image_size(path: str) -> int:
     return total_size
 
 
+def _get_cuda_version(path: str) -> dict:
+    """Get the cuda version."""
+    res = dict()
+    # gathering version from version.txt file
+    version_path = path + "/usr/local/cuda/version.txt"
+    if os.path.isfile(version_path):
+        with open(version_path, "r") as f:
+            for line in f.readlines():
+                if line.startswith("CUDA Version"):
+                    res["/usr/local/cuda/version.txt"] = line[len("CUDA Version"):].strip()
+                    _LOGGER.info("CUDA version identifier was found in file %s", res["/usr/local/cuda/version.txt"])
```

Comment:
Is it redundant?

Reply:
I don't think, so. We can see it in the logs.